src/Microsoft.NetCore.Analyzers/Core/Security/ReviewCodeForXssVulnerabilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.NetCore.Analyzers.Security.Helpers;
+
+namespace Microsoft.NetCore.Analyzers.Security
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class ReviewCodeForXssVulnerabilities : SourceTriggeredTaintedDataAnalyzerBase
+    {
+        internal static readonly DiagnosticDescriptor Rule = SecurityHelpers.CreateDiagnosticDescriptor(
+            "CA3002",
```

Comment:
Nit: Given the number of security rules being added, is it worth moving all the security diagnostic IDs into a separate file, similar to https://github.com/dotnet/roslyn-analyzers/blob/master/src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticIds.cs? Again, not something for this PR, but for future cleanup if you feel it helps.